src/services/refactors/convertArrowFunction.ts

```diff
@@ -0,0 +1,90 @@
+/* @internal */
+namespace ts.refactor.convertArrowFunction {

```

Comment:
I would have thought this would be a suggestion diagnostic/code-fix so that it could be discovered via the lightbulb.

Reply:
would not that make it a lint feature at this point?

Reply:
never mind.. all suggestions are lint features anyways..

Reply:
why this could be a diagnostic/code-fix...
I don't want to see this lightbulb in the code except when I want to convert